cypress/integration/parallel-6/savedArticles.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { featureSwitches } from '../../../shared/featureSwitches';
+
+if (!featureSwitches.savedArticles) {
+	describe('Feature Switch OFF: Saved Articles', () => {
+		beforeEach(() => {
+			cy.session('auth', () => {
+				cy.setCookie('gu-cmp-disabled', 'true');
+			});
+		});
+
+		it('redirects to account overview homepage from /saved-articles route ', () => {
+			cy.visit('/saved-articles');
+
+			cy.url().should('not.contain', '/saved-articles');
+
+			// check Navigation/Page Title
+			cy.findAllByText('Account overview').should(
+				'have.length.at.least',
+				1,
+			);
+			cy.findAllByText('Saved articles').should('have.length', 0);
+		});
+	});
+}
+
+if (featureSwitches.savedArticles) {
+	describe('Feature Switch ON: Saved Article', () => {
+		beforeEach(() => {
+			cy.session('auth', () => {
+				cy.setCookie('gu-cmp-disabled', 'true');
+			});
+		});
+
+		it('displays Saved Article page', () => {
+			cy.visit('/saved-articles');
+
+			cy.findAllByText('Saved articles').should('have.length', 2);
```

Comment:
Small point, but might be tempted to be more general here and say length greater than 0 for an existence check.

Reply:
I thought the length check would do that as well as indicate that the tab appears on the title AND LH navigation. I saw you added a comment in your PR which we can merge in 👍